odata-url-conventions/4 Resource Path.md

```diff
@@ -946,10 +946,12 @@ Requests to paths ending in `/$query` MUST use the `POST` verb. Query
 options specified in the request body and query options specified in the
 request URL are processed together.
 
-The request body MUST use the content-type `text/plain`. It contains the
-query portion of the URL and MUST use the same percent-encoding as in
-URLs (especially: no spaces, tabs, or line breaks allowed) and MUST
-follow the syntax rules described in chapter Query Options.
+The request body MUST use the content-type `text/plain` or `application/x-www-form-urlencoded`.
+It contains the query portion of the URL.
+
+For content-type `text/plain`, the individual query options MUST be separated by `&` or newlines
```

Comment:
If we allow newlines as separator here, the reason for introducing `application/x-www-form-urlencoded` is gone. Shall we remove it?